system/jlib/jstatcodes.h

```diff
@@ -311,6 +311,11 @@ enum StatisticKind
     StNumMatchRightRowsMax,
     StNumMatchCandidates,
     StNumMatchCandidatesMax,
+    StTimeSoapcallDNS,                  // Time spent in DNS lookups for soapcalls
+    StTimeSoapcallConn,                 // Time spent in connect[+SSL_connect] for soapcalls
```

Comment:
Connect would be better than Conn (since this is seen by the user)

Reply:
Sure.  I was just trying to save some chars.